proto/panacea/oracle/v2/tx.proto

```diff
@@ -47,8 +48,9 @@ message MsgApproveOracleRegistration {
 // ApproveOracleRegistration defines for oracle registration approval
 message ApproveOracleRegistration {
   string unique_id = 1;
-  string target_oracle_address = 2;
-  bytes encrypted_oracle_priv_key = 3;
+  string approved_oracle_address = 2;
```

Comment:
We need to know the address of the approved oracle.

Reply:
In my opinion, if the signature is created through `oraclePrivateKey`, signature verification is possible without `approved_oracle_address`. wdyt?

Reply:
We need signature verification for transactions. This transaction is the address of the approver.

Reply:
Ah, okay. It's for transaction `GetSigners`. Thanks.

Reply:
I think `approver_address` or `approver_oracle_address` would be another good name for this variable.
Accordingly, we can use `approved_oracle_address` instead of `target_oracle_address`.

If you feel it can be confusing, I could suggest another choice.
- `by_address` or `by_oracle_address`
- `to_address` or `to_oracle_address`

If you like it, please choose what you like :)

Reply:
I think `approver_oracle_address` like good. :)

Reply:
fix: https://github.com/medibloc/panacea-core/pull/519/commits/f7c3bf22340709be83301f78269414820a34f981